mybatis.properties

```diff
@@ -0,0 +1,14 @@
+category=External Analysers
+description=Custom Rule for Check Risk SQL of Mybatis Mapper XML
+homepageUrl=https://github.com/donhui/sonar-mybatis
+archivedVersions=
+publicVersions=1.0.0
+
+defaults.mavenGroupId=org.sonarsource.plugins.mybatis
+defaults.mavenArtifactId=sonar-mybatis-plugin
+
+1.0.0.description=Initial Update Center release
+1.0.0.sqVersions=[6.7,7.9.1]
```

Comment:
You really want the end of this range to be `LATEST` rather than 7.9.1. That way, when 7.9.2 or 8.0 is released, your plugin will be marked compatible by default.